lib/postinst-migrations.sh

```diff
@@ -71,4 +71,21 @@ except Exception as e:
     print('Error while creating public user-config file: {}'.format(e))
 "
     fi
+    # Create an only series marker file if not present
+    # This is to ensure that the only-series contract check runs correctly
+    if [ ! -f "/var/lib/ubuntu-advantage/private/machine-token.json" ]; then
+        /usr/bin/python3 -c "
+from uaclient.files import machine_token
+try:
+    machine_token_file = machine_token.get_machine_token_file()
+    only_series = machine_token_file.only_series()
+        if only_series:
+            print(only_series)
+except Exception as e:
+    print('Error while checking only-series field on contract: {}'.format(e))
+"
+        if [ -n "$only_series" ]; then
```

Comment:
`$only_series` is not being defined anywhere, so this will always be empty string. So, again, the marker file never gets to be created.

Shouldn't we, instead of printing, create the marker file in the python code itself? Using `state_files.only_series_check_marker_file`?

Reply:
The current integration test goes from series to series, but it would be nice to add a manual test to cover upgrading from older versions (with and without the overlay to force the only series)